docs: init (#706)
revanced/revanced-manager: docs/0_prerequisites.md

> # 💼 Prerequisites
> ## 📝 Requirements
> - Device running Android 8 or higher
> - Full app patching capability doesn't work for device that reported as ARMv7
> 
> ## ⬇️ Installation
> - Download and install the [latest ReVanced Manager APK](https://github.com/revanced/revanced-manager/releases/latest)
> 
> ## ⏭️ What's next
> The next section will guide you through patching an application.
> 
> Continue: [🧩 Patching applications](1_patching-applications.md)

Reviewer: ```suggestion
# 💼 Prerequisites

## 📝 Requirements

- Device running Android 8 or higher
- Full app patching capability doesn't work for device that reported as ARMv7

## ⬇️ Installation

- Download and install the [latest ReVanced Manager APK](https://github.com/revanced/revanced-manager/releases/latest)

## ⏭️ What's next

The next section will guide you through patching an application.

Continue: [🧩 Patching applications](1_patching-applications.md)
```

> # 💼 Prerequisites
> 
> ## 📝 Requirements
> 
> - Device running Android 8 or higher
> - Full app patching capability doesn't work for device that reported as ARMv7
> 
> ## ⬇️ Installation
> 
> - Download and install the [latest ReVanced Manager APK](https://github.com/revanced/revanced-manager/releases/latest)
> 
> ## ⏭️ What's next
> 
> The next section will guide you through patching an application.
> 
> Continue: [🧩 Patching applications](1_patching-applications.md)

Reviewer: ```suggestion
# 💼 Prerequisites

## 📝 Requirements

- Android 8+
- Any ABI other than ARMv7

## ⬇️ Installation

- Download and install the latest version of [ReVanced Manager](https://github.com/revanced/revanced-manager/releases/latest)

## ⏭️ What's next

The next section will guide you through patching an application.

Continue: [🧩 Patching applications](1_patching-applications.md)
```
